### ISSUE TITLE
Update package name to be clear about the API it conforms to?

### ISSUE THREAD

**Issue by BigLep** [libp2p/js-http-v1-content-routing#5]
It's great that we have this JS implementation.  A few things about the name...

This conforms to https://github.com/ipfs/specs/blob/main/routing/ROUTING_V1_HTTP.md right?  As a result...
   1. Can we get that into the README?
   2. Name it js-http-routing-v1?

Stretch: https://github.com/ipfs/specs/blob/main/routing/ROUTING_V1_HTTP.md involves more than just peer routing (the libp2p problem).  As a result, does this maybe belong in the ipfs github org?

In summary, I'm proposing we rename/move to ipfs/js-http-routing-v1 (or something like this) and update the README for clarity.

**Comment by achingbrain** [comment]
This is an implementation of the [content routing interface](https://libp2p.github.io/js-libp2p-interfaces/interfaces/_libp2p_interface_content_routing.ContentRouting.html) that uses Routing v1 HTTP (previously know as "reframe"), so it should correctly be called `routing-v1-http-content-routing` but I think `http-v1-content-routing` is something a human being might reasonably say instead?

**Comment by BigLep** [comment]
Sounds good.  So rename repo to `ipfs/js-http-v1-content-routing`?

**Comment by achingbrain** [comment]
This module has been renamed and published to https://www.npmjs.com/package/@libp2p/http-v1-content-routing - the previous version has a deprecation notice pointing to the new version - https://www.npmjs.com/package/@libp2p/reframe-content-routing
